React/Base/RCTBatchedBridge.m

```diff
@@ -116,7 +119,7 @@ - (void)start
     sourceCode = source;
     dispatch_group_leave(initModulesAndLoadSource);
   } onProgress:^(RCTLoadingProgress *progressData) {
-#ifdef RCT_DEV
+#if RCT_DEV && __has_include("RCTDevLoadingView.h")
```

Comment:
This used to check if the macro is defined, which it always is, so I made it consistent with the rest of the code base (which is to check its value).